src/networkAddresses.ts

```diff
@@ -199,4 +199,14 @@ export const NETWORK_ADDRESSES = {
     CHAINLINK_ORACLE: "0x493C3f543AEa37EefF17D823f27Cb1feAB9f3143",
     POOL_REGISTRY: "0x6538C861C7A6997602311342657b9143dD9E8152",
   },
+  opmainnet: {
+    NORMAL_TIMELOCK: "", // To be deployed
+    VTREASURY: "0x104c01EB7b4664551BE6A9bdB26a8C5c6Be7d3da",
+    GUARDIAN: "0x2e94dd14E81999CdBF5deDE31938beD7308354b3",
+    RESILIENT_ORACLE: "0x21FC48569bd3a6623281f55FC1F8B48B9386907b",
+    CHAINLINK_ORACLE: "0x1076e5A60F1aC98e6f361813138275F1179BEb52",
+    BOUND_VALIDATOR: "0x37A04a1eF784448377a19F2b1b67cD40c09eA505",
```

Comment:
We don't use/need the bound validator contract in the framework. I would remove it from here and I would define it in the multisig TX where it's used

Reply:
fixed, https://github.com/VenusProtocol/vips/pull/386/commits/0e3d5904e56f8af9f8844016daa585a02c256a51